config/Dockerfiles/Jenkinsfile

```diff
@@ -34,15 +34,19 @@ OPENSHIFT_SERVICE_ACCOUNT = env.OPENSHIFT_SERVICE_ACCOUNT ?: 'jenkins'
 executionID = UUID.randomUUID().toString()
 
 // Pod name to use
-podName = 'linchpin_container-' + executionID
+podName1 = 'lp_container-1-' + executionID
+podName2 = 'lp_container-2-' + executionID
+podName3 = 'lp_container-3-' + executionID
 
-library identifier: "ci-pipeline@master",
-        retriever: modernSCM([$class: 'GitSCMSource',
-                              remote: "https://github.com/CentOS-Paas-SIG/ci-pipeline"])
+def libraries = ['cico-pipeline'           : ['master', 'https://github.com/CentOS/cico-pipeline-library.git'],
+                 'ci-pipeline'             : ['master', 'https://github.com/CentOS-PaaS-SIG/ci-pipeline.git'],
+                 'upstream-fedora-pipeline': ['smalltest', 'https://github.com/johnbieren/upstream-fedora-pipeline.git']]
```

Comment:
If you +1 this:
https://github.com/CentOS-PaaS-SIG/ci-pipeline/pull/716
You can remove this shared library and change the following lines from
packagepipelineUtils.clearTemplateNames()
to
pipelineUtils.clearTemplateNames()
Lines 100, 133, 166